src/modules/Admin/components/ListBooking/ListBooking.tsx

```diff
@@ -0,0 +1,285 @@
+    setBookingId(id);
+    setOpenDelete(true);
+  };
+
+  // Modal styling
+  const style = {
+    position: "absolute" as "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 500,
+    bgcolor: "background.paper",
+    border: "none",
+    boxShadow: 24,
+    borderRadius: 2,
+    p: 4,
+  };
+
+  // Get all bookings
+  const getBooking = async () => {
+    try {
+      const response = await axios.get(
+        `${bookingUrl.getAllBooking}?page=${page}&limit=${rowsPerPage}`,
+        {
+          headers: { Authorization: localStorage.getItem("token") || "" },
+        }
+      );
+      setBooking(response.data.data.booking);
+      setTotalCount(response.data.data.totalCount); // Set total count
+    } catch (error: any) {
+      toast.error(error.response?.data?.message || "Failed to fetch bookings");
+    }
+  };
+
+  // Delete booking
+  const deleteBooking = async () => {
+    try {
+      const response = await axios.delete(bookingUrl.delete(bookingId), {
+        headers: { Authorization: localStorage.getItem("token") || "" },
+      });
+      toast.success(response.data.message || "Deleted Successfully");
+      getBooking();
+      handleCloseDelete();
+    } catch (error: any) {
+      toast.error(error.response?.data?.message || "Failed to delete booking");
+    }
+  };
+
+  // Style rows (color alternation)
+  const StyledTableRow = styled(TableRow)`
+    &:nth-of-type(even) {
+      background-color: #f8f9fb;
+    }
+    font-family: "Poppins" !important;
+  `;
+
+  // Pagination handlers
+  const handleChangePage = (_event: unknown, newPage: number) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  useEffect(() => {
+    getBooking();
+  }, [page, rowsPerPage]);
+
+  return (
+    <>
+      {/* Delete Modal */}
+      <Modal
+        open={openDelete}
+        onClose={handleCloseDelete}
+        aria-labelledby="delete-modal-title"
+        aria-describedby="delete-modal-description"
+        sx={{ fontFamily: "Poppins", padding: "60px" }}
+      >
+        <Box sx={style}>
+          <Box sx={{ display: "flex", justifyContent: "end", mb: "60px" }}>
+            <i
+              onClick={handleCloseDelete}
+              style={{ color: "#CC0000", cursor: "pointer" }}
+              className="fa-regular fa-xl fa-circle-xmark"
+            ></i>
+          </Box>
+
+          <Box sx={{ textAlign: "center", mt: 5, mb: 2 }}>
+            <img src={DeleteImg} alt="Delete" />
+          </Box>
+
+          <Typography
+            variant="h6"
+            sx={{ color: "#494949", fontWeight: 700, textAlign: "center" }}
+          >
+            Delete This Booking?
+          </Typography>
+          <Typography
+            sx={{
+              color: "#494949",
+              fontSize: "14.5px",
```

Comment:
it is not recommended to use pixels for font-sizes, you can read more about that here:
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility